owncloudApp/src/main/res/values/strings.xml

```diff
@@ -673,9 +673,29 @@
     <string name="release_notes_proceed">Proceed</string>
     <string name="release_notes_icon">Release note icon</string>
 
-    <!-- Release Notes 2.21.1 -->
+    <!-- Release Notes 2.21.1_oem -->
+    <string name="release_notes_title_2_21_1_oem_1">Release Notes</string>
+    <string name="release_notes_subtitle_2_21_1_oem_1">News in the app will be listed after every update to inform about latest improvements and new features. Available also in Settings view to recheck.</string>
 
-    <string name="release_notes_title_2_21_1_1">Crash in \"Open with\" operation</string>
-    <string name="release_notes_subtitle_2_21_1_1">Crash fixed in details view when operation \"Open with\" is selected.</string>
+    <string name="release_notes_title_2_21_1_oem_2">Secure connections by default</string>
+    <string name="release_notes_subtitle_2_21_1_oem_2">Https connections by default to establish server connections. Otherwise, it will be explicitly required in URL</string>
+
+    <string name="release_notes_title_2_21_1_oem_3">Passcode and Pattern redesign</string>
+    <string name="release_notes_subtitle_2_21_1_oem_3">Integrated the numeric keypad on the screen, improved navigation and graphical interface.</string>
+
+    <string name="release_notes_title_2_21_1_oem_4">Support for SVG files</string>
+    <string name="release_notes_subtitle_2_21_1_oem_4">Added support for svg files.</string>
+
+    <string name="release_notes_title_2_21_1_oem_5">Details view redesign</string>
+    <string name="release_notes_subtitle_2_21_1_oem_5">Information reordered and added the possibility to download and open the file by clicking the thumbnail.</string>
+
+    <string name="release_notes_title_2_21_1_oem_6">Hidden files</string>
+    <string name="release_notes_subtitle_2_21_1_oem_6">Added a new option in settings to decide to show or not hidden files.</string>
```

Comment:
Some subtitles have a final `.` and some don't. Should we remove the final dot from every entry? What do you think?

Reply:
you're right. i will remove the final dots.

Reply:
Done